src/dswx_sar/refine_with_bimodality.py

```diff
@@ -108,8 +104,8 @@ def __init__(self,
                                   self.prob,
                                   expected,
                                   bounds=(
-                                    (-30, 0, 0,
-                                    -30, 0, 0),
+                                    (-30, 1e-10, 0,
+                                    -30, 1e-10, 0),
```

Comment:
I think you meant a negligibly small number by `1e-10`. Can this be replaced by `dswx_sar_util.Constants.negligible_value`?